vdbench.py

```diff
@@ -1,367 +1,367 @@
-        Arguments (str, str, int): file_name,vd_name,diskindex
-        Return (str, str): workload_path, result_path
-        '''
-        uri = "servers"
-        res = ILDC().do_ssy_details(uri, header=None)
-        self.build = res.json()[0]['ProductBuild']
-        workload_path = os.path.abspath(r'../../../Config/VdBench_config/Workload')
-        self.absulute = os.path.abspath(r'../../../Result/Vdbench')
-        self.new_ = self.absulute + '/'+self.build+'_'+self.time_stamp+'/'
-        if file_name.split('.')[0] == '4-4k-4-fill':
-            result_path = self.new_+vd_name+'/'+workload+'_'+file_name.split('.')[0]
-        else:
-            result_path = self.new_+vd_name+'/'+vd_name+'_'+file_name.split('.')[0]
-        workload_path = os.path.join(workload_path, file_name)
-        file = open(workload_path, "r+")
-        data = file.readlines()
-        file.close()
-        for index,val in enumerate(data):
-            split_ = val.split('PhysicalDrive')
-            if len(split_) > 1:
-                data[index] = split_[0] + "PhysicalDrive" + str(diskindex)+"\n"
-        file = open(workload_path, "w")
-        for _ in data:
-            file.write(_)
-        file.close()
-        return workload_path, result_path
-
-class ResultCreation():
-    '''
-    Class:- ResultCreation
-    This class is going to collect VdBench result and store in
-    HTML formate.
-    Arguments : None
-    Return: None
-        '''
-    glob_flag = 0
-    data_put = []
-    destiny = ''
-    path = ''
-    result_path = ''
-    merge_list = []
-    zfs_max = ''
-    build = '' 
-    def get_server(self):
-        '''
-        This method collect all results of SSY required to update
-        in HTML file
-        Arguments : None
-        Return: None
-        '''
-        uri = "servers"
-        res = ILDC().do_ssy_details(uri, header=None)
-        self.build = res.json()[0]['ProductBuild']
-        host = res.json()[0]['HostName']
-        ram = res.json()[0]['TotalSystemMemory']['Value']
-        ram_ = str(round(int(ram)/1073741824,2))
-        ram_ =ram_ + ' GB'
-        available = res.json()[0]['AvailableSystemMemory']['Value']
-        sync = res.json()[0]['IldcConfigurationData']['IldcSyncMode']
-        primaycach = res.json()[0]['IldcConfigurationData']['IldcPrimaryCacheMode']
-        ssy = int(ram) - int(available)
-        ssy = (round((ssy)/1073741824,2))
-        ssy = str(ssy) + ' GB'
-        if self.merge_list[0] != '-':
-            zfs = int(self.zfs_max)/1073741824
-            zfs = str(round(zfs,2))
-            zfs = zfs + ' GB'
-        else:
-            zfs = '-'
-        self.data_put = [self.build, host,str(zfs),
-                         str(ssy), primaycach, ram_ , sync, '500GB']
-    def run(self):
-        '''
-        This method create dynamic folder structure to store results
-        Arguments : None
-        Return: None
-        '''
-        self.get_server()
-        configur = ConfigParser()
```

Comment:
Again, any object that you create should be like config_object for better readability